m4opt/models/bandpass/bandpass.py

```diff
@@ -0,0 +1,202 @@
+from astropy.modeling import Model
+from astropy.modeling.models import Tabular1D
+from astropy import units as u
+from astropy.units.physical import get_physical_type
+
+from .core import Background
+
+# TODO: remove check below when tynt is released
+# and add tynt to required dependencies
+try:
+    from tynt import FilterGenerator
+    has_tynt = True
```

Comment:
Remove `has_tynt` flag; no need for it because instead can just check if `tynt_filters is None`.